src/compiler/factory.ts

```diff
@@ -1530,9 +1537,8 @@ namespace ts {
     }
 
     export function createStatement(expression: Expression) {
-        const node = <ExpressionStatement>createSynthesizedNode(SyntaxKind.ExpressionStatement);
-        node.expression = parenthesizeExpressionForExpressionStatement(expression);
-        return node;
+        return createExpressionStatement(parenthesizeExpressionForExpressionStatement(expression));
+

```

Comment:
Nit: extra newline.

Reply:
We could consider linting for [`no-padding`](https://github.com/Microsoft/dtslint/blob/master/src/rules/noPaddingRule.ts).